### ISSUE TITLE
Support via Monday instead of Zammad?

### ISSUE THREAD

**Issue by Matzeroth** [corpatch/daily-business#6]
There is the idea to organize RMA Process through monday. This might be most effective if also ticket system (right now in Zammad) may be done with monday. 

Question: Is money a funktional alternative to zammad?

**Comment by Sascha-Herrmann** [comment]
yes, Monday do also have a ticketsystem. But we should first understand what (databases, Hetzner server, etc) is related to Zammad.

**Comment by Matzeroth** [comment]
O.K. Nur zur Info: Emergency Calls können reduziert werden. Hier RS mit Kornelius die Schwelle z.B. 300 Sekunden zu setzten. 

Und: Die Löschung auf Zammad ist durch uns programmiert und kann jederzeit gelöscht werden. DSGVO wäre zu klären ob wir die immer behalten dürften.

**Comment by Matzeroth** [comment]
Die Funktionalitäten von Zammad sollten automatisiert in Monday übernommen werden. 

- Auto Response an Kunden
- Automatisches Erstellen von Tickets
- Automatisches Zuordnen von Tickets
- Zuordnung der Tickets ändern
- Automatisches wiedereröffnen von Tickets
- Tags zur statistischen Auswertung an Tickets anhängen
- Verknüpfung mit RMA Datenbank
- Interne Wissensdatenbank zur Beantwortung der Tickets
- Textbausteine zur Beantwortung der Fragen
